contents/data_vis_makie_themes.md

```diff
@@ -26,6 +26,7 @@ s = """
         with_theme(demo_themes, theme_minimal())
         with_theme(demo_themes, theme_light())
     ] # hide
+   #link_attributes = "width=60%" # hide
```

Comment:
Ah, yes I haven't implemented broadcasting `Options` for anything except filename.

You can manually create an `Vector{Options}` with something like
```
[Options(fig; link_attributes) for fig in objects]
```

Reply:
I tried without success :( . I will tried another day.